source/globus/cli.rst

```diff
@@ -58,15 +58,18 @@ You can find the names of all VSC collections in our table of :ref:`globus-avail
 You can list the contents of a collection with 'globus ls', followed by the collection id.
 You can specify a directory path on the collection after a colon::
 
+      # Making a variable for the collection ID for convenience
+      $ kuleuventier2scratch=872a58ab-02b9-4233-a3e0-f017ed8ab090
+
       # To list a collection from the root: 
-      $ globus ls $tier2scratch        
+      $ globus ls $kuleuventier2scratch        
             nproject/
             nscratch/
             project/
             scratch/                                                            
       
       # To list a specific directory:
-      $ globus ls $tier2scratch:scratch/337/vsc33731/test
+      $ globus ls $kuleuventier2scratch:scratch/<redacted>/<username>/test
```

Comment:
What this "redacted" here means? The number that is used on the path? It is this explained somewhere???

Reply:
Indeed unclear, changed it to 'xxx' and 'vscxxxxx' and added a comment about it.